src/containers/login.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react"
+import styled from "styled-components"
+
+import FacebookButton from "../components/buttons/facebook"
+import Button from "../components/buttons/inverted"
+import TwitterButton from "../components/buttons/twitter"
+import Icon from "../components/icon"
+import Input from "../components/input"
+import Text from "../components/text"
+import TextLink from "../components/text_link"
+
+interface LoginProps extends React.Props<HTMLParagraphElement> {
+  form?: {
+    url: string,
+    csrfToken?: string,
+  },
+  onSubmit?: () => void
+}
+
+interface LoginState {}
+
+const LoginContainer = styled.div`
+  padding: 20px;
+  max-width: 350px;
+  margin: 0 auto;
+  align-text: center;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+`
+
+const StyledInput = styled(Input)`margin: 5px 0`
```

Comment:
would be nice if there's a consistent set of margin/padding values we can add to a component since the vast majority of custom CSS comes from the need of specifying custom margins/paddings. Something like [basscss-margin](http://basscss.com/#basscss-margin) but styled component version.